internal/connect/utils.go

```diff
@@ -10,6 +10,10 @@ import (
 	"github.com/confluentinc/cli/v3/pkg/errors"
 )
 
+var (
+	BYOCSupportClouds = []string{"aws", "azure"}
```

Comment:
Please define this in `pkg` instead of `internal`, this is a standard practice in Go to avoid import cycles